extensions/counter/src/main/resources/benchmark-clustered-counters.xml

```diff
@@ -0,0 +1,59 @@
+<!-- RadarGun 3.0 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}" />
+
+   <clusters>
+      <cluster size="2" />
+   </clusters>
+
+   <configurations>
+      <config name="Infinispan 9.0 - Strong Clustered Counter">
+         <setup plugin="infinispan90">
+            <!-- You can set JVM arguments for slaves here. -->
+            <vm-args>
+               <memory max="4G" />
+            </vm-args>
+            <embedded xmlns="urn:radargun:plugins:infinispan90:3.0" file="clustered-counters.xml"/>
+         </setup>
+      </config>
+     
+   </configurations>
+
+   <rg:scenario xmlns:rg="urn:radargun:benchmark:3.0"
+                xmlns="urn:radargun:stages:core:3.0"
+                xmlns:counter="urn:radargun:stages:counter:3.0">
+
+      <service-start/>
+
+      <jvm-monitor-start />
+
+      <!-- other options for operation name: DECREMENT_AND_GET, ADD_AND_GET, COMPARE_AND_SET -->
+      <counter:counter-test test-name="incrementAndGetTest"
+                              num-operations="1000"
+                              num-threads-per-node="1"
+                              operation-name="INCREMENT_AND_GET"
+                              counter-name="c1"
+                              initial-value="0">
+      </counter:counter-test>
+
+      <!-- Due to a bug in RG we need to add num-threads-per-node * cluster size -->
```

Comment:
what bug?

Reply:
In Stressor#runInternal the the following method is run once at the beginning and later during normal execution. ```logic.run(operation)```
I'm not sure why this behavior is there but it means that each stressor calls the operation n+1 times (the additional call is IMO a bug)

Reply:
It is not run once; there are two loops: first executes requests (but not recording them) until stage.isStart() returns true, that is until all threads start (and the main thread observes that).

So it seems that in your case the main thread manages to kick in immediately after the first wait-for-others operation, but it is not guaranteed.

Reply:
OK. What is the reason to call the logic.run on some threads before all threads are started?

Reply:
If all threads started at once, there could be convoying due to concurrent conflicts etc... We should probably do even more of this ramp-up to randomize the times when threads send their requests.